modules/installing-gcp-querying-labels-tags-gcp.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: REFERENCE
+[id="installing-gcp-querying-labels-tags-gcp_{context}"]
+= Querying user-defined labels and tags for GCP
+
+After creating the {product-title} cluster, you can access the list of the labels and tags defined for the GCP resources by referring to the new fields, `.status.platformStatus.gcp.resourceLabels` and `.status.platformStatus.gcp.resourceTags` present in the `infrastructures.config.openshift.io/cluster` object as shown in the following sample `infrastructure.yaml` file.
+
+.Sample `infrastructure.yaml` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Infrastructure
+metadata:
+ name: cluster
+spec:
+ platformSpec:
+   type: GCP
+status:
+ infrastructureName: <cluster_id><1>
+ platform: GCP
+ platformStatus:
+   gcp:
+     resourceLabels:
+     - key: <label_key>
+       value: <label_value>
+     resourceTags:
+     - key: <tag_key_short_name>
+       parentID: <OrganizationID/ProjectID>
+       value: <tag_value_short_name>
+   type: GCP
+----
+<1> The value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`.
```

Comment:
This doesn't make sense. This simply calling out the field name and nothing about what it is.

Reply:
How about " Contains the value of `.status.infrastructureName` present in `infrastructures.config.openshift.io/cluster`."

Reply:
The point here is to have callout mention what the field is about. The current statement is redundant.

Reply:
Confirmed: "The cluster ID is the value that is generated at the cluster installation."